topydo/ui/Main.py

```diff
@@ -150,6 +158,39 @@ def hide_viewwidget():
         self.column_mode = _APPEND_COLUMN
         self._set_alarm_for_next_midnight_update()
 
+    def _create_color_palette(self):
+        project_color = _to_urwid_color(config().project_color())
```

Comment:
We have to insert `if config().colors()` here as soon as we get rid of that override above.